### PR TITLE
#2496 Publish nightly doc

### PR DESCRIPTION
**Pull request by barondim**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

**Comment by barondim** [comment]
Do we need to build all documentation ? (build-doc, build-doxygen-doc and build-nrt-doc ?) @gautierbureau

**Comment by barondim** [comment]
- [ ] put zip files on github

**Comment by gautierbureau** [comment]
You need to do this in nightly-release.yml instead

File: .github/workflows/nightly-release.yml [hunk 1]
```diff
@@ -148,12 +153,17 @@ jobs:
...
+          pushd distributions
           for FILE in *zip; do
             [ -f "$FILE" ] || break
             echo "Upload asset $FILE"
             upload_zip_file $UPLOAD_URL $FILE ${{ secrets.GITHUB_TOKEN }}
           done
+          popd
+
+          pushd documentation/dynawoDocumentation
+          upload_pdf_file $UPLOAD_URL DynawoDocumentation.pdf ${{ secrets.GITHUB_TOKEN }}
```
Comment (rosiereflo) [inline .github/workflows/nightly-release.yml:165]: We could also publish the tools example documentation?

**Comment by barondim** [comment]
Currently the pdflatex package is missing on the Fedora container.

- [ ] Waiting for the installation of the pdflatex package

File: .github/workflows/nightly-release.yml (hunk 1 above)
Comment (barondim) [reply .github/workflows/nightly-release.yml:165]: We won't be able to pull that off for this PR unfortunately.

Comment (rosiereflo) [reply .github/workflows/nightly-release.yml:165]: why is that?

**Review by gautierbureau — APPROVED** [review]

**Comment by rosiereflo** [comment]
@gautierbureau can we merge this one?

**Pull request merged**